robottelo/virtwho_utils.py

```diff
@@ -8,9 +8,12 @@
 from robottelo.cli.host import Host
 from robottelo.cli.virt_who_config import VirtWhoConfig
 from robottelo.config import settings
+from robottelo.config.virtwho import VirtwhoSettings
 from robottelo.constants import DEFAULT_ORG
 
 VIRTWHO_SYSCONFIG = "/etc/sysconfig/virt-who"
+virtwhosettings = VirtwhoSettings()
```

Comment:
Could we change this to a short name, such as:
`virtwho = VirtwhoSettings()`

Reply:
Changed. And also tested all cases, updated the test result in the description